mate-panel/libpanel-util/panel-dbus-service.c

```diff
@@ -54,6 +51,12 @@ static void panel_dbus_service_get_property (GObject    *object,
 					     guint       prop_id,
 					     GValue     *value,
 					     GParamSpec *pspec);
+void
+panel_dbus_service_on_signal (GDBusProxy *proxy,
```

Comment:
This is a declaration of a function which doesn't exist. It does nothing as is. 

@yetist: did you plan to implement that function as a replacement for the removed `panel_dbus_service_on_name_owner_changed`?